src/components/MemberList.js

```diff
@@ -0,0 +1,125 @@
+import { KEY, PRIORITY } from '../constants.js';
+import { addMemberButtonTemplate, memberTemplate } from '../templates.js';
+
+export default class MemberList {
+  constructor({
+    onAddMember,
+    onFilterTodoList,
+    onDeleteTodoList,
+    onAddTodoItem,
+    onDeleteTodoItem,
+    onToggleTodoItem,
+    onUpdateTodoItem,
+    onUpdateTodoItemPriority,
+  }) {
+    this.$team = document.querySelector('.todoapp-list-container');
+
+    this.$team.addEventListener('click', (event) => this.addMember(event, onAddMember));
+    this.$team.addEventListener('click', (event) => this.filterTodoList(event, onFilterTodoList));
+    this.$team.addEventListener('click', (event) => this.deleteTodoList(event, onDeleteTodoList));
+    this.$team.addEventListener('keydown', (event) => this.addTodoItem(event, onAddTodoItem));
+    this.$team.addEventListener('click', (event) => this.deleteTodoItem(event, onDeleteTodoItem));
+    this.$team.addEventListener('click', (event) => this.toggleTodoItem(event, onToggleTodoItem));
+    this.$team.addEventListener('dblclick', (event) => this.editTodoItem(event));
+    this.$team.addEventListener('keydown', (event) => this.updateTodoItem(event, onUpdateTodoItem));
+    this.$team.addEventListener('change', (event) => this.updateTodoItemPriority(event, onUpdateTodoItemPriority));
+  }
+
+  render(members) {
+    const template = members.map(memberTemplate);
+    this.$team.innerHTML = template.join('') + addMemberButtonTemplate;
+  }
+
+  addMember(event, onAddMember) {
+    const addMemberButtonTarget = event.target.closest('button');
+    if (!addMemberButtonTarget) return;
+    if (addMemberButtonTarget.id !== 'add-user-button') return;
+    onAddMember();
+  }
+
+  filterTodoList(event, onFilterTodoList) {
+    const filterButtonTarget = event.target;
+    if (filterButtonTarget.tagName !== 'A') return;
+
+    const [filterName] = filterButtonTarget.classList;
+    onFilterTodoList(filterButtonTarget.id, filterName);
+  }
+
+  deleteTodoList(event, onDeleteTodoList) {
```

Comment:
```suggestion
  deleteTodoList({ target }, onDeleteTodoList) {
```
위의 코드처럼 필요하신 target을 추출해낼 수 있습니다!

Reply:
event에서 target만 사용하기 때문에 이런 식으로 작성해도 좋겠네요~! 감사합니다